explainability/explainability-core/src/main/java/org/kie/kogito/explainability/local/lime/HighScoreNumericFeatureZones.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.kogito.explainability.local.lime;
+
+import java.util.stream.DoubleStream;
+
+public class HighScoreNumericFeatureZones {
+
+    private final double[] highFeatureScorePoint;
+    private final double tolerance;
+
+    public HighScoreNumericFeatureZones(double[] highFeatureScorePoint, double tolerance) {
+        this.highFeatureScorePoint = highFeatureScorePoint;
+        this.tolerance = tolerance;
+    }
+
+    public boolean accept(double point) {
+        return DoubleStream.of(highFeatureScorePoint).anyMatch(d -> point > (d - tolerance) && point < (d + tolerance));
+    }
+}
```

Comment:
Do you expect this class to have additional behaviours in the future? Because at the moment it looks like a `Predicate`

```suggestion
public class HighScoreNumericFeatureZones implements Predicate<Double> {

    private final double[] highFeatureScorePoint;
    private final double tolerance;

    public HighScoreNumericFeatureZones(double[] highFeatureScorePoint, double tolerance) {
        this.highFeatureScorePoint = highFeatureScorePoint;
        this.tolerance = tolerance;
    }

    @Override
    public boolean test(Double point) {
        return DoubleStream.of(highFeatureScorePoint).anyMatch(d -> point > (d - tolerance) && point < (d + tolerance));
    }
}
```

Reply:
thanks, it makes sense to convert it to a `Predicate`